src/Data/Form.purs

```diff
@@ -151,6 +161,74 @@ userInput = case _ of
   UserInput x -> Just x
   _ -> Nothing
 
+newtype Errors e = Errors (Set e)
```

Comment:
I'm not sure `Set e` is the most appropriate type to model a collection of errors. The idea was that since it's a Set, you wouldn't be able to have duplicate errors (e.g. as opposed to an array, where you could have `[Required, Required]`). 

But if my `e` is something like `data ValidationErrors = Required | MaxLength Int`, I can still have `fromFoldable (MaxLength 10 : MaxLength 20 : Nil)`, though that should be pretty hard to get from how our forms work.

Still, it's the best I could come up with for modeling them 🤷‍♂️

Reply:
Seems right to me!